.circleci/config.yml

```
@@ -1,6 +1,86 @@
 version: 2.1
 
 commands:
+  conditionally_exclude_changes:
```

This might communicate the behavior more clearly:

```suggestion
  halt_if_unchanged_except:
```